src/lib/tests/bitcoin/bitcoin-chain-transfers.test.ts

```diff
@@ -4,7 +4,7 @@ import * as backendApiFactory from '../../bitcoin/bitcoin-backend-api'
 import { chainTransfersApiFactory } from '../../transfers';
 import { currency } from '../helpers';
 
-const backendApi = backendApiFactory.withCurrency("http://backendApiUrl", currency)
+const backendApi = backendApiFactory.withCurrency("http://backendApiUrl", currency, () => '')
```

Comment:
może dac default parameter że ten corr id będzie `() => ''`?

Reply:
Niby można, ale wolę, żeby developer świadomie to wpisał, jeśli faktycznie chce mieć pustą wartość